src/Function/Metric/Nat/Bundles.agda

```diff
@@ -15,12 +15,14 @@ module Function.Metric.Nat.Bundles where
 open import Data.Nat.Base hiding (suc; _⊔_)
 open import Function.Base using (const)
 open import Level using (Level; suc; _⊔_)
-open import Relation.Binary.Core
-open import Relation.Binary.PropositionalEquality
-  using (_≡_; isEquivalence)
+open import Relation.Binary.Core using (Rel)
+open import Relation.Binary.PropositionalEquality.Core using (_≡_)
+open import Relation.Binary.PropositionalEquality.Properties using (isEquivalence)
 
-open import Function.Metric.Nat.Core
+open import Function.Metric.Nat.Core using (DistanceFunction)
 open import Function.Metric.Nat.Structures
+  using (IsProtoMetric; IsPreMetric; IsQuasiSemiMetric; IsSemiMetric; IsMetric;
+         IsUltraMetric)
```

Comment:
I feel this isn't a very useful narrowing of the import. `Bundles` is always going to want to import everything in `Structures`....